src/internal/modules/invoice-accounts/controllers/contacts.js

```diff
@@ -84,8 +87,49 @@ const postContactCreate = async (request, h) => {
   return h.postRedirectGet(form, urlJoin('/invoice-accounts/create/', regionId, companyId, 'create-contact'));
 };
 
+const getContactSearch = async (request, h) => {
```

Comment:
I think this should be called getSearchCompany and move to the invoice-accounts controller because you are searching for a company in the database and not a contact.

Reply:
OK. I will move this.